package.json

```diff
@@ -28,7 +28,7 @@
     "build": "tsc -b",
     "lint": "run-p lint:*",
     "lint:es": "cross-env PARSER_NO_WATCH=true eslint . --cache --ext js,md.mdx,ts,tsx -f friendly",
-    "lint:tsc": "tsc --noEmit",
+    "lint:tsc": "tsc",
```

Comment:
removed no emit due to
```
tsconfig.json:4:5 - error TS5053: Option 'noEmit' cannot be specified with option 'incremental'.

4     "noEmit": true
      ~~~~~~~~


Found 1 error.
```

Reply:
We can use `tsc --incremental=false --noEmit`

Reply:
updated in tsconfig.json